### ISSUE TITLE
digest lost images in some feed.

### ISSUE THREAD

**Issue by mantis3d** [tanriol/digest#81]
this is "http://songshuhui.net" feed in Brief

![brief](https://f.cloud.github.com/assets/2351096/1207418/5711e46e-25b7-11e3-868f-56f92103f85f.png)

and this is same one in Digest

![digest](https://f.cloud.github.com/assets/2351096/1207424/81020ad8-25b7-11e3-9957-6ef3aba8581b.png)

where are those images??

**Comment by tanriol** [comment]
What Digest version do you use? Works for me in 1.1.0

By the way, as Adam came back and integrated most Digest fixes into Brief, Digest users are advised to migrate back to Brief.

**Comment by mantis3d** [comment]
I use 1.1.0 too. 
Only a few feeds lost image, for example "http://portableapps.com" and "http://songshuhui.net", others are working fine.

**Comment by tanriol** [comment]
Hmm. Could you please open Digest with such a feed, right-click where the image should be, choose "Inspect element" and check whether there is an `<img src="...">` tag and the address of the image?

**Comment by mantis3d** [comment]
Yes, there is a IMG tag, but it's not show, may be the position is wrong?

![digest-b](https://f.cloud.github.com/assets/2351096/1214976/3c8c0198-2651-11e3-812a-e6ef260ab6f8.png)

and this is same one in Brief
![brief-b](https://f.cloud.github.com/assets/2351096/1214977/3cbb7d38-2651-11e3-82a2-55b418683fb2.png)

**Comment by tanriol** [comment]
Hmm. I don't see anything bad, which is... pretty strange.

By the way, running Digest and Brief simultaneously in the same Firefox instance is definitely not recommended. They use the same database and do not synchronize with each other, so races and other consistency issues may occur.

**Comment by ghost** [comment]
Same problem here with various feeds from comic sites. Examples are:
http://abstrusegoose.com/feed
https://xkcd.com/rss.xml

&lt;img&gt; tags are there in the source but the images are not shown. In Brief everything works properly. I'm using Digest 1.1.0.

**Comment by tanriol** [comment]
Works for me with Digest 1.1.0 for xkcd.com at least. Are there any other addons that may cause this problem?

**Comment by ghost** [comment]
> Are there any other addons that may cause this problem?

This seems to be the problem indeed. I just tried with a clean profile that has only Digest installed and the images show up correctly there. Sorry I did not think of this in the first place. However, Brief has no problems in my main Firefox profile...

I'll investigate this further and let you know when I find the offending/incompatible extension.

**Comment by mantis3d** [comment]
Well, I found that is the addon "RequestPolicy" cause the problem, after I turn on "Temporarily allow all requests", all images are showed. Or I add the feed web url to the destinations whitelist, images in this feed are showed. but I can't add the Digest to Origins whitelist. Why does Digest has this problem, but Brief is not?

**Comment by tanriol** [comment]
Digest and Brief use different means for feed display. As the following Digest versions, if any, will be based on Brief, this bug will go away.

**Comment by ghost** [comment]
> I found that is the addon "RequestPolicy" cause the problem

Confirmed! Thanks for testing, I simply forgot it...

> As the following Digest versions, if any, will be based on Brief

Any estimates on when a new version is expected?
